lib/utils/logger.js

```diff
@@ -0,0 +1,12 @@
+var _ = require('lodash');
```

Comment:
[Note] This file may just strictly define standard for logging. We must probably expose this as separate module (It's small but it provides some common functionality.